### ISSUE TITLE
Section numbers missing in docx reader + html writer

### ISSUE THREAD

**Issue by KaptenLutman** [jgm/pandoc#10233]
Heading numbers in Word are correctly rendered in html for figures, tables, equations, and table of contents, but they are missing in the headings themselves.

command:
pandoc -s -t html5+smart -o html/test.html --log=html/log.txt -f docx+native_numbering+styles+citations --verbose --css=idms.css --toc-depth=5 --mathjax --metadata lang=en-US --resource-path=media test.docx

Manually edited html file to get the desired web fonts and table layouts together with the css file (included in attached archive).

pandoc version: 3.4
Windows version:
Edition	Windows 11 Pro
Version	23H2
Installed on	‎2024-‎09-‎10
OS build	22631.4169
Experience	Windows Feature Experience Pack 1000.22700.1034.0

[test.docx](https://github.com/user-attachments/files/17173903/test.docx)
[test.zip](https://github.com/user-attachments/files/17173930/test.zip)

**Comment by jgm** [comment]
We don't carry over section or figure numbers from the docx. Rather, the writer generates these numbers. If you want section numbers, use the `--number-sections` option.

**Comment by jgm** [comment]
Pandoc's `--number-sections` just affects the renderer.  The docx reader won't pay attention to the numbers, it just gets the section structure.

**Comment by KaptenLutman** [comment]
I used Word's automatic numbering (chapter number as prefix) also for other numbered items, such as figures, tables, equations, and those chapter numbers were correctly rendered. It looks like the chapter numbers are ignored only for the headers. Interestingly, the results didn't change when I removed the `--number-sections` option. Is there any way to instruct pandoc to render chapter numbers starting from something other than 1? It's just the highest level that is wrong - all other numbers are correct. Anyway, I could solve the problem by manually search and replace `section-number">1` with `section-number">3` and `data-number="1` with `data-number="3`
